CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## icu4x 1.3.3 (later Oct, 2023)
+
+Some crates received a 1.3.3 patch release:
+
+- [Fix #4165, locale "vi" not working in Collator](https://github.com/unicode-org/icu4x/issues/4165):
+    - `icu_collator`
+    - `icu_collator_data`
+- [Fix #4232, some locale data being improperly deduplicated](https://github.com/unicode-org/icu4x/issues/4232):
```

Comment:
can we be more specific? what data exactly did this affect?

Reply:
People can click the bug link for more details

Reply:
not clear from the bug for me. It mentions es-MX, but the diff includes other languages as well

Reply:
I added a bit more info in the bug.